api/src/services/users.js

```diff
@@ -48,11 +48,14 @@ const SETTINGS_EDITABLE_FIELDS = RESTRICTED_SETTINGS_EDITABLE_FIELDS.concat([
 
 const META_FIELDS = ['token_login'];
 
+// No longer used, but allowed for backwards compatibility
+const LEGACY_FIELDS = ['language'];
+
 const ALLOWED_RESTRICTED_EDITABLE_FIELDS =
   RESTRICTED_SETTINGS_EDITABLE_FIELDS.concat(RESTRICTED_USER_EDITABLE_FIELDS, META_FIELDS);
 
 const illegalDataModificationAttempts = data =>
-  Object.keys(data).filter(k => !ALLOWED_RESTRICTED_EDITABLE_FIELDS.includes(k));
+  Object.keys(data).filter(k => !ALLOWED_RESTRICTED_EDITABLE_FIELDS.concat(LEGACY_FIELDS).includes(k));
```

Comment:
I think a change got lost in the merge here, removing `language` from `RESTRICTED_SETTINGS_EDITABLE_FIELDS`. Right now, you end up editing the language after all.

Reply:
Wow.  I really wish I had done a merge instead of a rebase now.  (Would have made it easier to figure out what happened here. I don't remember any conflicts while rebasing, but apparently something got messed up...)  Anyway, I have removed 'language' again from the RESTRICTED_SETTINGS_EDITABLE_FIELDS and have read back through all the other changes looking for anything else that got dropped (but did not find anything).

Reply:
Yea :) 
A unit test should have caught this.